### PR TITLE
🎨 UI updates - 🐛 bug fixes

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?

🎨 UI updates
Button and color updates - (_This will be ongoing. I still want to handle the input fields and other colours as things progress_)

🐛 bug fixes

Dashboard new design: small refinements

1. Hovering on sharing at the bottom of the page is trimmed - **Added space at the bottom** 
2. Tutorials tab, it seems that the user is selecting something - **Fixed the problem**
3. Click to select project is a bit broken - Needs a proper solution
4. Remove "Dashboard" text when it is not actionable - **Done**

## Related issue/s

o²S²PARC UX/UI Improvements #1083 ?
fixes [Dashboard new design: small refinements #5086](https://github.com/ITISFoundation/osparc-simcore/issues/5086)

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/dashboard/GridButtonItem.js [hunk 1]
```diff
@@ -101,22 +105,46 @@ qx.Class.define("osparc.dashboard.GridButtonItem", {
...
+          control.getContentElement().setStyles({
+            "border-radius": `${this.self().MENU_BTN_DIMENSIONS / 2}px`
+          });
           this._add(control, {
-            top: 4,
-            right: 4
+            top: 8,
+            right: 8
           });
           break;
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/dashboard/GridButtonItem.js:149]: Looks like we are recreating the study card with a different look everytime it gets selected/unselected or the selection mode is activated/deactivated. I would rather change the state of the existing card without recreating it. Then the card could have three states: standard, unselected and selected, and change its own look accordingly

**Review by ignapas — COMMENTED** [review]
just a couple of comments, I could not actually try the thing yet

File: services/static-webserver/client/source/class/osparc/navigation/NavigationBar.js [hunk 2]
```diff
@@ -314,7 +315,7 @@ qx.Class.define("osparc.navigation.NavigationBar", {
     _applyPageContext: function(newCtxt) {
       switch (newCtxt) {
         case "dashboard":
-          this.getChildControl("dashboard-label").show();
+          this.getChildControl("dashboard-label").exclude();
```
[the remark this answers is not in the document]
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/navigation/NavigationBar.js]: This is intended

**Review by matusdrobuliak66 — APPROVED** [review]

**Review by elisabettai — APPROVED** [review]
Looks good to me and in my opinion it fixes the linked issue.

There's only the "Click to select project is a bit broken" that needs a proper fix, as far as I understand, but maybe it is not too critical. Where you thinking to fix that in another PR at some point?

**Review by sanderegg — COMMENTED** [review]
looks great!

**Comment by jsaq007** [comment]
> can we use a blue checkmark (in the last screenshot) instead of black

@ignapas The blue circle was intended to properly show it as selected. Would you be OK with this till other feedback comes in?

File: services/static-webserver/client/source/class/osparc/dashboard/GridButtonItem.js (hunk 1 above)
Comment (ignapas) [reply services/static-webserver/client/source/class/osparc/dashboard/GridButtonItem.js]: no, not a show stopper. let me run the code

**Comment by ignapas** [comment]
Is this still a draft? Let me know when I should review. Thanks.

**Comment by jsaq007** [comment]
> Is this still a draft? Let me know when I should review. Thanks.

Sorry forgot to update the PR :)

**Review by ignapas — COMMENTED** [review]
This is looking great, already an amazing improvement!
Just a couple of minor details, I guess not "musts" but I think it would be nice to get in with the rest:

**In the light mode**
* Hovering of some buttons
Screenshot from 2023-12-07 15-28-39
Screenshot from 2023-12-07 15-27-40
Screenshot from 2023-12-07 15-30-12

And that's it! Thanks a lot...

**Comment by jsaq007** [comment]
> This is looking great, already an amazing improvement! Just a couple of minor details, I guess not "musts" but I think would be nice to get in with the rest:
> 
> **In the light mode**
> 
> * When hovering some buttons, the text/icons have almost no contrast
> 
> Screenshot from 2023-12-07 15-28-39 Screenshot from 2023-12-07 15-27-40 Screenshot from 2023-12-07 15-30-12
> 
> And that's it! Thanks a lot...

Hi @ignapas these should be handled a little better?

Screenshot 2023-12-11 at 09 54 39
Screenshot 2023-12-11 at 09 55 14
Screenshot 2023-12-11 at 09 55 22

@elisabettai I also handle the submit service button in here as I missed in my previous PR
https://app.marker.io/i/6565d3d3b40e96cdb74e9d5b_532944bf9137aca0?activePane=issue 
Screenshot 2023-12-11 at 09 58 04

**Review by ignapas — APPROVED** [review]
👍🏼

**Pull request merged**